modules/QuickEncounter.js

```diff
@@ -377,7 +377,7 @@ export class QuickEncounter {
         //0.7.0b: Capture controlled tiles (will be one or the other
         const controlledTiles = Array.from(Tile.layer.controlled);
 
-        const candidateJournalEntry = QuickEncounter.findCandidateJournalEntry();
+        const candidateJournalEntry = (this instanceof JournalEntry ? this : QuickEncounter.findCandidateJournalEntry()); //Allow runAddOrCreate to be passed a Journal Entry to use instead of searching for one.
```

Comment:
Changing this because further down I'm getting Quick Encounters to add a button to the Enhanced Journals header.  This button is telling Quick Encounters to specifically use a Journal Entry.